### PR TITLE
Add notes about style for emphasis and span of code

### PR DESCRIPTION
**Pull request by rgaiacs**

### CODE REVIEW

File: _episodes/06-style-guide.md [hunk 1]
```diff
@@ -10,6 +10,86 @@ keypoints:
...
+
+  ~~~
+  we want to output the lines that do not contain the word *the*.
+  ~~~
+
+  This is preferable instead of the use of quotation marks.
+
+  See the "Keyboard Key" section for characters or keyboard keys that the learn should type.
+  
+- for words and phrases that are still regarded as foreign.
```
Comment (gcapes) [inline _episodes/06-style-guide.md]: >regarded as *unfamiliar* 

might be better?

File: _episodes/06-style-guide.md [hunk 2]
```diff
@@ -10,6 +10,86 @@ keypoints:
...
+- for titles of books, periodicals, plays, films, TV, radio series, and music albums.
+
+  For example,
+
+  ~~~
+  we will use a file that contains three haikus taken from a 1998 competition in *Salon* magazine.
```
Comment (gcapes) [inline _episodes/06-style-guide.md]: Capitalisation 
>**W**e will use a file ...

File: _episodes/06-style-guide.md [hunk 3]
```diff
@@ -10,6 +10,86 @@ keypoints:
 - "Keyboard keys need to use `<kbd>` HTML tag."
 ---
 
+## Emphasis
+
+Markdown treats asterisks (_*_) as indicators of emphasis.
```
Comment (gcapes) [inline _episodes/06-style-guide.md]: I would suggest changing to 
>... indicators of emphasis, and renders text marked up like this in *italics*.

File: _episodes/06-style-guide.md [hunk 4]
```diff
@@ -10,6 +10,86 @@ keypoints:
...
+We use emphasis
+
+- to highlight a word, phrase, or character where it is itself the object of discussion.
+
+  For example,
+
+  ~~~
+  we want to output the lines that do not contain the word *the*.
```
Comment (gcapes) [inline _episodes/06-style-guide.md]: Capitalise first letter of sentence

File: _episodes/06-style-guide.md [hunk 5]
```diff
@@ -10,6 +10,86 @@ keypoints:
...
+
+  For example,
+
+  ~~~
+  we will use a file that contains three haikus taken from a 1998 competition in *Salon* magazine.
+  ~~~
+
+## Strong Emphasis
+
+Markdown treats double asterisks (_**_) as indicators of strong emphasis.
```
Comment (gcapes) [inline _episodes/06-style-guide.md]: I would suggest changing to 
> ... indicators of strong emphasis, and renders text marked up like this in **boldface**

File: _episodes/06-style-guide.md [hunk 6]
```diff
@@ -10,6 +10,86 @@ keypoints:
...
+
+  ~~~
+  you can run it by opening a terminal and typing `bash`.
+  ~~~
+
+  When the object of discussion is the program or language in a broad sense,
+  we don't use span of code. For example,
+
+  ~~~
+  The most popular Unix shell is Bash
```
Comment (gcapes) [inline _episodes/06-style-guide.md]: Full stop required.

File: _episodes/06-style-guide.md [hunk 7]
```diff
@@ -10,6 +10,86 @@ keypoints:
...
+  ~~~
+- to highlight one function where it is itself the object of discussion.
+
+  For example,
+
+  ~~~
+  `len` is much faster than any function we could write ourselves
```
Comment (gcapes) [inline _episodes/06-style-guide.md]: Full stop

**Comment by rgaiacs** [comment]
This will close https://github.com/swcarpentry/lesson-example/issues/105.

**Comment by gcapes** [comment]
Thanks @rgaiacs for pursuing this. I think one thing that is currently missing(?) is how best to mark up file and directory names.

**Pull request merged**